typescript-react/src/components/Label/Label.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import FormControl from 'react-bootstrap/FormControl';
+import ControlLabel from 'react-bootstrap/FormLabel';
+
+import { I18nContext } from '../I18n/I18n';
+import { localizeText } from '../I18n/service';
+
+interface ICustomisableProps {
```

Comment:
actually, since this wasn't yet released, please fix all this british english

Reply:
Sure thing, PR is open #178